src/patternfly/layouts/Toolbar/styles.scss

```diff
@@ -2,18 +2,29 @@
 
 // The value of component scoped variables is always defined by a global variable:
 .pf-l-toolbar {
+  // Toolbar Layout
+  // adds negative margin to toolbar to offset the top margin given on all group divs
+  margin-top: calc(var(--pf-l-toolbar__group--MarginTop) * -1);
```

Comment:
You want to keep all of the CSS vars at the top of the selector, and properties that apply to the selector at the bottom (after any css vars). Also, this is a `margin-top` on `.pf-l-toolbar` but your var is `--pf-l-toolbar__group--MarginTop`. You should use a variable named `--pf-l-toobar--MarginTop` here.

Reply:
this portion of the code has been corrected. ready for next push.